integreat_compass/cms/templates/components/report_list.html

```diff
@@ -0,0 +1,46 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    {% translate "Reports" %}
+{% endblock title %}
+{% block content %}
+    {% include "includes/dashboard_navbar.html" %}
+    {% for report in reports %}
+        <div class="flex flex-row border-2 border-gray-200 bg-white rounded-lg shadow-md shadow-inner p-6 mb-8">
+            <div class="basis-2/3 pr-8">
+                <p class="text-xl font-bold">{{ report.offer_version.title }}</p>
+                <p>{{ report.offer_version.offer.organization.org_name }}</p>
+                <p class="text-xl text-gray-700 font-semibold pt-4">{% translate "Short description" %}</p>
+                <p class="text-gray-600">{{ report.offer_version.description|truncatechars:500 }}</p>
+                <div class=" my-4">
+                    {% for tag in report.offer_version.offer.tags.all %}
+                        <span class="bg-red-500 text-white rounded-full py-1 px-2">{{ tag }}</span>
+                    {% endfor %}
+                </div>
+                <p class="text-gray-700">{% translate "Created at:" %} {{ report.offer_version.created_at|date:"d. M Y" }}</p>
+            </div>
+            <div class="basis-1/3 flex flex-col justify-around align-center">
+                <div>
+                    <h3 class="font-semibold text-gray-700">{% translate "Reason" %}</h3>
+                    <p class="text-gray-600">{{ report.comment }}</p>
+                </div>
+                <form action="{% url 'cms:protected:reports' %}" method="post">
+                    {% csrf_token %}
+                    <input type="hidden"
+                           name="offer_version_id"
+                           value="{{ report.offer_version.id }}">
+                    <div class="grid lg:grid-cols-3 grid-cols-1 xl:gap-8 gap-2 text-white ">
+                        <button class="bg-blue-500 hover:bg-blue-700 rounded-full" type="button">{% translate "Details" %}</button>
+                        <button class="bg-red-500 hover:bg-red-700 rounded-full"
+                                type="submit"
+                                name="move-to-declined"
+                                value="True">{% translate "Reject version" %}</button>
+                    </div>
+                </form>
+            </div>
+        </div>
+    {% endfor %}
```

Comment:
```suggestion
    {% empty %}
        {% translate "There are no unhandled reports." %}
    {% endfor %}
```

(or something along the lines)